### PR TITLE
Mango chicken

### PR DESCRIPTION
**Pull request by henningonsbring**
### This PR adds | fixes:
Good food

**How to prepare for test**:
- [ ] `ssh` to ...
- [ ] Install on stage:
`bash servers/resources/SERVER.scilifelab.se/update-[THIS_TOOL]-stage.sh [THIS-BRANCH-NAME]`

### How to test:
-

### Expected outcome:
- [ ]

### Review:
- [ ] Code approved by
- [ ] Tests executed by
- [ ] "Merge and deploy" approved by

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: docs/dinner/mango_chicken [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+Add the creme fraiche, mango chutney and sambal oelek, then let it slightly boil for a few minutes
+
+Start boiling the rice in a separat caserol
+
+Chop the paprika and mango into pieces
+
+At the end add the paprika and mango
+
+Serve with indian beer like Cobra!
```
Comment (Karl-Svard) [inline docs/dinner/mango_chicken:26]:
```suggestion
Serve with indian beer like Cobra!

```

Comment (Karl-Svard) [reply docs/dinner/mango_chicken:26]: Add newline to end of file.

File: docs/dinner/mango_chicken [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Mango chicken by Henning
+
+![Mango_chicken](https://www.dinneratthezoo.com/wp-content/uploads/2018/06/mango-chicken-4.jpg)
+
+## Ingredients
```
Comment (Karl-Svard) [inline docs/dinner/mango_chicken:5]:
```suggestion
## Ingredients

```

Comment (Karl-Svard) [reply docs/dinner/mango_chicken:5]: Add newline after header.

File: docs/dinner/mango_chicken [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+- 1 Mango
+- 500 g Chicken
+- Rice
+- Mango chutney (half of a can)
+- 5 dl creme fraiche
+- 3 msk soya
+- 2 tsk sambal oelek
+- 1 red paprika
+
+## Preparation
```
Comment (Karl-Svard) [inline docs/dinner/mango_chicken:15]:
```suggestion
## Preparation

```

**Review by Karl-Svard — COMMENTED** [review]
Small fixes

**Comment by henningonsbring** [comment]
I recommend everyone to try this recipe despite the PR getting closed now without merging to master branch.

**Pull request closed without merging**